bin/export-record-barcodes

```diff
@@ -38,6 +38,12 @@ BARCODE_FIELDS = [
     *[f'barcode_optional_{i}' for i in range(1,5)],
     "core_collection_barcode",
     "return_collection_barcode",
+    *[f'welcome_barcode_{i}' for i in range(1,3)],
+    *[f'serial_barcode_{i}' for i in range(1,5)],
+    *[f'return_serial_barcode_{i}' for i in range(1,5)],
+    "outgoing_barcode",
+    "barcode",
```

Comment:
Do you happen to know if there was a motivation for these particular variable names for apple project's kit registration and pcdeqc forms? I'm not super keen on the project's redcap introducting yet another variant of the same variable name if there's not a reason to do so.

Reply:
I do have some context about a couple of these variable, because I asked Ashley about them.
'outgoing_barcode' = The barcode sent out in the replacement illness kit.
'collection_barcode' = This is on the PCDEQC form.

Ashley's answer: "I used the variable names that your template told me to you." And she is absolutely correct.
From https://redcap.iths.org/redcap_v10.5.2/Design/online_designer.php?pid=23589&page=core_fields
collection_barcode: Previously return_utm_barcode or sfs_barcode
outgoing_barcode: previously pre_scan_barcode

So, we should either update our project template to use variable names that we want, or we should update our code to accommodate the variable names we're asking people to use.

The template does not include the barcode variable to use when a PT registers a kit. This is missing from the template. For this one, Ashley chose 'barcode'.

Reply:
@kschwabefry , in REDCap we renamed 'barcode' to 'core_activation_barcode', so I've made that change in here. I also saw that I had forgotten to add the Apple barcodes to derived-tables.sql, so I added them in the Apple commit too.

Reply:
Thanks Chris, this makes a lot of sense. I like the new name core_activiation_barcode.